testing-utils/src/main/java/no/sikt/nva/data/report/testing/utils/ResultSorter.java

```diff
@@ -27,12 +27,12 @@ private ResultSorter() {
         // NO-OP
     }
 
-    public static String sortResponse(ReportFormat type, String data, String sortByHeader1, String sortByHeader2)
+    public static String sortResponse(String type, String data, String sortByHeader1, String sortByHeader2)
```

Comment:
String? Is this done on purpose?

Reply:
Didn't want to leak report-api model into other modules. But an enum would be better, will change type 👍